Gemfile

```diff
@@ -6,8 +6,6 @@ gem 'bootsnap', '>= 1.1.0', require: false # Reduces boot times through caching;
 gem 'bunny', '~> 2.19' # for rabbitmq
 gem 'committee' # Validates HTTP requests/responses per OpenAPI specification
 gem 'config'
-gem 'dry-monads', '~> 1.3'
```

Comment:
not sure why dry-monads is explicitly mentioned here;  assume it's leftover from dor_indexing gem extraction